### PR TITLE
Prep release 2.7.2

### PR DESCRIPTION
**Pull request by jordanch**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+    "requires": true,
+    "dependencies": {
+        "@types/prop-types": {
+            "version": "15.7.3",
+            "resolved": "https://registry.npmjs.org/@types/prop-types/-/prop-types-15.7.3.tgz",
+            "integrity": "sha512-KfRL3PuHmqQLOG+2tGpRO26Ctg+Cq1E01D2DMriKEATHgWLfeNDmq9e29Q9WIky0dQ3NPkd1mzYH8Lm936Z9qw==",
+            "dev": true
+        },
+        "@types/react": {
+            "version": "16.9.50",
```
Comment (IIsaku) [inline package-lock.json:14]: Aren't we on React 17 now for native with MX 9?

Comment (jordanch) [reply package-lock.json:14]: We are, however the changes in `package-lock` are from running `npm i` and committing the file. No changes to `package.json` were made. I'd like to not touch the dependencies for now.

Comment (jordanch) [reply package-lock.json:14]: The reason being is that this is a whole other tuna can - Atlas 2 and run on MX 8 + 9

Comment (jordanch) [reply package-lock.json:14]: "Can of fish" - get it ;)

**Review by IIsaku — APPROVED** [review]

**Pull request merged**